project/Dockerfile

```diff
@@ -8,6 +8,8 @@ WORKDIR /usr/src/app
 RUN python -m pip install --upgrade pip
 COPY ./requirements.txt .
 RUN pip install -r requirements.txt
```

Comment:
As described <a href="https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better"> Here</a> it is better to combine your `RUN` statements into one command using `&&\\n` because each RUN statement translates to an additional image layer in the image which increases the disk space required to store the Docker Image.

My second consternation is described in part of the answer to the above SO:
```
I may pull a collection of common utilities to most of those images into the first run command 
so the other images benefit from caching.
```
Since we are using GitHub actions Cache v2 to cache the docker image layers I think that you should take advantage of the cache by combining all of the RUN statements into one. For the short term while the `requirements .txt` is updated often that will require rebuilding of that layer but additions to the requirements are going to drop off at some point and caching will be used to save from re-downloading the NLTK modules repeatedly. 

for these reasons can you change the following code to
```Dockerfile
RUN pip install -r requirements.txt && \
python -m nltk.downloader punkt && \
python -m nltk.downloader averaged_perceptron_tagger
```

or if you want to separate the layers based on expected churn rate

```Dockerfile
RUN pip install -r requirements.txt

RUN python -m nltk.downloader punkt && \
python -m nltk.downloader averaged_perceptron_tagger
```

Reply:
Thanks for the thorough comment! I have consolidated into a single `RUN` statement in the latest commit.